### PR TITLE
Added Theme Color Red and Added Separate Stylesheet for CSS

### PR DESCRIPTION
**Pull request by NaviRocker**
- Added Theme Color Red

- Separate CSS Stylesheet
Solved #16

### CODE REVIEW

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM! 🚀 
It's always better to have one commit per PR. If you want you can try. It's always preferable to learn new things.

File: _site/css/style.css [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+a {
+	text-decoration: none;
+	color: #090;
+}
+
+a:hover {
+	text-decoration: underline;
+}
+#side-bar h1 {
+			display: block;
```
Comment (vaibhavsingh97) [inline _site/css/style.css]: This File has inconsistent indentaion. Please fix it.

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by NaviRocker** [comment]
@vaibhavsingh97 Now it's fine

**Comment by vaibhavsingh97** [comment]
@naveentherocker Please scheck, Still an issue

**Comment by NaviRocker** [comment]
Can you please mention the issue @vaibhavsingh97

File: _site/css/style.css [hunk 2]
```diff
@@ -0,0 +1,369 @@
+html, body {
```
Comment (dilpreetsio) [inline _site/css/style.css:1]: @naveentherocker Looks like the indentation is not consistent throughout this file. Please check

File: _site/css/style.css [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+    min-height: 100% !important;
+    height: 100%;
+}
+nav {
```
Comment (dilpreetsio) [inline _site/css/style.css]: Add a new line here.

File: _site/css/style.css [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+li {
+	display:inline-block;
+	float: left;
+	margin-right: 1px;
+}
+
+/*Style for menu links*/
+li a {
+	display:block;
+	min-width:140px;
```
Comment (dilpreetsio) [inline _site/css/style.css]: @naveentherocker For properties use this syntax:
```prop: value;```
eg.
```min-width: 140px;```
Make these changes throughout the file

Comment (NaviRocker) [reply _site/css/style.css]: Well I didn't add CSS styles. Everything you mentioned was already there. Ok I'll do those changes

**Comment by NaviRocker** [comment]
I think Everything is Fine now

**Review by isuruAb — APPROVED** [review]

**Pull request merged**